modules/hivemq/src/test/java/org/testcontainers/hivemq/docs/DemoHiveMQContainerIT.java

```diff
@@ -25,15 +25,15 @@ public class DemoHiveMQContainerIT {
 
     // hiveEEVersion {
     @Container
-    final HiveMQContainer hivemqEe = new HiveMQContainer(DockerImageName.parse("hivemq/hivemq-ce").withTag("2021.3"))
+    final HiveMQContainer hivemqEe = new HiveMQContainer(DockerImageName.parse("hivemq/hivemq4").withTag("4.7.4"))
         .withLogLevel(Level.DEBUG);
 
     // }
 
     // eeVersionWithControlCenter {
     @Container
     final HiveMQContainer hivemqEeWithControlCenter = new HiveMQContainer(
-        DockerImageName.parse("hivemq/hivemq-ce").withTag("2021.3")
+        DockerImageName.parse("hivemq/hivemq4").withTag("4.7.4")
```

Comment:
```suggestion
        DockerImageName.parse("hivemq/hivemq4").withTag("4.7.4")
```

We can also bump it to the latest version :)